### PR TITLE
use devsandboxci-aws profile for toolchain-e2e repo

### PR DESCRIPTION
**Pull request by MatousJobanek**
Let's change the profile in toolchain-e2e for now. If it works, then we will change it at other places in the next PR

### CODE REVIEW

File: ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml [hunk 1]
```diff
@@ -18,9 +18,8 @@ resources:
       memory: 250Mi
 tests:
 - as: e2e
-  skip_if_only_changed: ^doc/|\.md$|\.adoc$|^(?:.*/)?(\.gitignore|\.codecov\.ya?ml|\.yamllint|sonar-project\.properties|OWNERS|PROJECT|LICENSE)$
```
Comment (MatousJobanek) [inline ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml:21]: Let's drop this - it's not really useful and it brings more complications than benefits

**Review by mfrancisc — APPROVED** [review]

**Pull request merged**